### DISCUSSION TITLE
Issues when using drop down menus on certain PA plugins

### DISCUSSION THREAD

**Discussion by th3funnychurch3s** [apohl79/audiogridder#1219] [Help]
E.g. Amek compressor and Brainworx bx_limiter True Peak disappear momentarily when I click on the resize menu (top left). It reappears after a second or two, with no changes applied. Some drop down menus on the bx True Peak plugin (e.g. the loudness target) causes the same issue.

[Server.zip](https://github.com/apohl79/audiogridder/files/11466159/Server.zip)
[Plugin.zip](https://github.com/apohl79/audiogridder/files/11466160/Plugin.zip)

**Comment by th3funnychurch3s** [comment]
The same thing happens with the new Brainworx bx_masterdesk PRO

**Comment by apohl79** [comment]
created an issue ticket for this, thx for reporting
